fix: drafonfly to valkey migration guide (#1806)
aiven/aiven-docs: docs/products/valkey/howto/migrate-dragonfly-to-valkey.md

- **Enable the module on Valkey**: In the [Aiven Console](https://console.aiven.io/),
  go to your Valkey service **Service settings** > **Advanced configuration** and
  enable the required module before loading data.
- **Transform the data**: Extract module-type values and re-insert them using native
  Valkey data types (for example, store JSON as a plain string with `SET`).
- **Skip the keys**: If the data is not critical, exclude those keys from migration.

### Filter keys (optional)

To exclude specific keys or key patterns from the migration, use a script to scan
and pipe only the keys you need. See
[Step 2](#step-2-migrate-data-to-aiven-for-valkey) for the scan-based approach.

## Step 2: Migrate data to Aiven for Valkey

Choose one of the following methods based on your use case.

> ### Method A: Live key-by-key migration using SCAN and DUMP/RESTORE

Reviewer: ```suggestion
### Method A: Key-by-key migration using SCAN and DUMP/RESTORE
```

> Use this approach to migrate a subset of keys or to keep both services in sync

Reviewer: ```suggestion
Use this approach to migrate all or a subset of keys.
```

   the Valkey target. Both Aiven services require TLS:

   ```toml
   [scan_reader]
   address = "<dragonfly-host>:<dragonfly-port>"
   password = "<dragonfly-password>"

>    tls = true

Reviewer: let's add one line to this file:
```
  count    = 500  # keys per SCAN iteration; raise for throughput
```
